react/features/base/react/components/native/Switch.js

```diff
@@ -0,0 +1,2 @@
+/* @flow */
```

Comment:
@hristoterezov, empty line after /* flow */ or not? You have the two in this PR. Anyway I'd advise:

````javascript
// @lfow

...
````